packages/embark-blockchain-connector/src/provider.js

```diff
@@ -5,6 +5,7 @@ const fundAccount = require('./fundAccount');
 const constants = require('embark-core/constants');
 const Transaction = require('ethereumjs-tx');
 const ethUtil = require('ethereumjs-util');
+const { transaction, sign } = require('@omisego/omg-js-util');
```

Comment:
We already use `ethereumjs-tx` to sign transactions as you can see here: https://github.com/embark-framework/embark/blob/f6947fc0823524e8b81aa774af77c7441dec0da7/packages/embark-blockchain-connector/src/provider.js#L120
You probably can just reuse that or switch to omg's one, so that we only use one lib

Reply:
The `@omisego/omg-js-util` library contains functions that sit on top of `ethereumjs-util`, and in order to pull this out, we'd need to duplicate what this library offers in our own code, which seems superfluous IMO. In other words, this library signs using the same underlying dependencies to sign an object (`ethereumjs-util.ecsign`), but offers some additional convenience in terms of building signable EIP-712 typed data.

Reply:
Ah ok, do you think you can use the omg library for the other ones then and we could get rid of the ethereumjs-tx lib. If you prefer, it could be done in another PR

Reply:
I believe the omg lib doesn't wrap `Transaction`, but will double check before saying for sure.

Reply:
Yep, can confirm `@omisego/omg-js-util` only has `ethereumjs-util` as a dependency, and not `ethereumjs-tx`. What do you think we should do in this case?

Reply:
It's fine, I hadn't looked into it, we can keep both then since they have different use cases